### PR TITLE
test(sbb-icon): add visual tests

### PR DESCRIPTION
**Pull request by DavideMininni-Fincons**

### CODE REVIEW

File: src/elements/icon/icon.visual.spec.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { html } from 'lit';
```
Comment (jeripeierSBB) [inline src/elements/icon/icon.visual.spec.ts:1]: looking at icon.scss, tests which could be added are color mutations, and testing color-immutable. And if it's possible a test with "data-empty" (no icon loaded). If it's too time consuming we can save it for another improvement ;)

**Review by jeripeierSBB — APPROVED** [review]
LGTM, some idea for improvements

File: src/elements/icon/icon.visual.spec.ts (hunk 1 above)
Comment (DavideMininni-Fincons) [reply src/elements/icon/icon.visual.spec.ts:1]: I can easily add a test with color=red and a test with no icon name; 
for the color-immutable in the call stack from 'loadSvgIcon' to 'validateContent' i see no way to do it programmatically

File: src/elements/icon/icon.visual.spec.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  describeViewports({ viewports: ['small'] }, () => {
+    for (const size of ['small', 'medium', 'large']) {
+      it(
+        `icon=circle-information size=${size} ${visualDiffDefault.name}`,
+        visualDiffDefault.with(async (setup) => {
+          await setup.withFixture(html`<sbb-icon name="circle-information-${size}"></sbb-icon>`);
+        }),
+      );
+
+      it(
```
Comment (jeripeierSBB) [inline src/elements/icon/icon.visual.spec.ts:17]: You could take out the color test from the size loop as color is not dependant on size

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**